core/cache/modelsummarywatcher_test.go

```diff
@@ -221,6 +221,44 @@ func (s *modelSummaryWatcherSuite) TestRemoveModelRemovesModel(c *gc.C) {
 	})
 }
 
+func (s *modelSummaryWatcherSuite) TestModelAnnotationsChange(c *gc.C) {
+	watcher := s.controller.WatchAllModels()
+	defer workertest.CleanKill(c, watcher)
+
+	changes := watcher.Changes()
+	// discard the initial event
+	_ = s.next(c, changes)
+
+	s.ProcessChange(c, cache.ModelChange{
+		ModelUUID: "model-2-uuid",
+		Name:      "model-2",
+		Life:      life.Alive,
+		Owner:     "bob",
+		UserPermissions: map[string]permission.Access{
+			"bob":  permission.AdminAccess,
+			"mary": permission.ReadAccess,
+		},
+		Annotations: map[string]string{
+			"muted": "true",
+		},
+	}, s.events)
```

Comment:
This test doesn't clearly indicate that only the Annotations are different between before and after.

Reply:
Made more clear.